src/plugins/region_map/server/services/utils/constants.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export const DEFAULT_HEADERS = {
+  'Content-Type': 'application/json',
+  Accept: 'application/json',
```

Comment:
```suggestion
  'Accept': 'application/json',
```

Reply:
Had added quotes, the lint checker removed it : )